nextcloudappstore/core/models.py

```diff
@@ -56,18 +56,12 @@ def can_delete(self, user: User) -> bool:
 
     @staticmethod
     def search(lang, terms):
```

Comment:
I think the Django way would be to create a manager class and add that method to the manager.